### ISSUE TITLE
Jump arrows don't appear if viewer is rendered to hidden element

### ISSUE THREAD

**Issue by sixlettervariables** [AlanDeSmet/human-resource-machine-viewer#4]
I'm not sure if this is a Chrome thing, or a styling issue for the SVG, but I'm not seeing any jump arrows.

Example: https://s3.amazonaws.com/christopherwatford-com/hrm/hrmfiddle-test.html

**Comment by AlanDeSmet** [comment]
Doesn't work in Firefox either, so I guess at least it's consistent. :-)

Without looking deeper, I'm guessing that because the &lt;table&gt; is hidden at the point the arrows are created, the arrows can't get the positional information they need and it pretty much just freaks out. I'm going to pull down a copy and do some quick tests.  If this is the case, the answer may be to expose another function "rerender_arrows"or similar to force them to recalculate.

**Comment by AlanDeSmet** [comment]
Right, I'm pretty sure it's the table thing. Upon reconsideration, rerender_arrows requires maintaining state, and at the moment I don't. It's a feature I want to add (to support things like a visible program counter), but not yet, and probably not in the next week.

All I can offer for a workaround is to hold off on calling append_code_table until the View tab is opened.

**Comment by sixlettervariables** [comment]
I've done just that and it works fine.
